doc/developer_information.rst

```diff
@@ -5,57 +5,59 @@ Reproducible Builds
 *******************
 
 This section describes how to reproduce the official released binaries for
-restic for version 0.10.0 and later. For restic versions down to 0.9.3 please
+restic for version 0.12.1 and later. For restic versions down to 0.9.3 please
```

Comment:
The GOPATH also shows up in the binaries at least since restic 0.10.0. So let's keep that old version number here. (The updated version number for the example is fine).

Reply:
Done.